.eslintignore

```diff
@@ -0,0 +1,4 @@
+public/*
```

Comment:
These ignores were previously inlined in the CLI.  Makes using `eslint` cli more consistent in behavior no matter where it is invoked.